packages/utils/src/misc.ts

```diff
@@ -25,13 +25,19 @@ export function uuid4(): string {
   const gbl = GLOBAL_OBJ as typeof GLOBAL_OBJ & CryptoGlobal;
   const crypto = gbl.crypto || gbl.msCrypto;
 
-  if (crypto && crypto.randomUUID) {
-    return crypto.randomUUID().replace(/-/g, '');
+  let getRandomByte = (): number => Math.random() * 16;
+  try {
+    if (crypto) {
+      if (crypto.randomUUID) {
```

Comment:
```suggestion
    if (crypto && crypto.randomUUID) {
```

Should work the same and be a bit nicer to read...?

Reply:
nvm sorry